### PR TITLE
✨ Source Lever Hiring: Migrate to Low-code CDK

### LINKED ISSUE #28115: Lever Hiring

### PR DESCRIPTION
**Pull request by topefolorunso**
## What
Migrating Source Lever Hiring to Low-Code resolves #28115

## How
Created the connector using the connector buider UI, then imported the manifest.yaml to the low code cdk.

## Recommended reading order
1. `manifest.yaml`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+    http_method: GET
+    request_headers: {}
+    authenticator:
+      class_name: source_lever_hiring.components.AuthenticatorLeverHiring
+      basic_auth: "#/definitions/basic_authenticator"
+      oauth2: "#/definitions/oauth2_authenticator"
+    request_body_json: {}
+  opportunity_child_stream:
+    type: DeclarativeStream
+    name: '{{ parameters.name }}'
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/manifest.yaml:82]: It looks like this is causing some streams to be called {{ parameters.name }} when we run `discover`.

I'll solve this in my next commit by adding the extra key-value pairs the old-fashioned way under each child stream's `$ref`: https://docs.airbyte.com/connector-development/config-based/advanced-topics/#references

**Comment by topefolorunso** [comment]
@sh4sh Any update on this PR please?

**Comment by marcosmarxm** [comment]
Hello, and thank you so much for contributing to the project during the Hackathon. Due to the high complexity of the code migration, we still have a few contributions open. I'm sending this message to update you that this contribution will be added to the final points we're going to announce tomorrow. With that said, we ask for your help to continue the process and finish the review.

**Comment by sh4sh** [comment]
Upon review, the original Python version of this connector uses a dynamic schema, which isn't currently possible to implement with the low code framework. Unfortunately I have to close this pull request for now, but we can look at it again onc we're able to implement it in low code.. Thanks for your efforts @topefolorunso and sorry that we can't merge this.

**Pull request closed without merging**